pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "ipykernel", # used by demo_dlmbl
     "graphviz", # used by demo_dlmbl
     "torchview", # used by demo_dlmbl
+    "pathlib",
```

Comment:
Pathlib is a [standard library](https://docs.python.org/3/library/pathlib.html).
```suggestion
```

Reply:
Ah, deleting.